core/dbt/task/compile.py

```diff
@@ -89,16 +90,22 @@ def task_end_messages(self, results):
             matched_results = []
 
         for result in matched_results:
-            fire_event(
-                CompiledNode(
-                    node_name=result.node.name,
-                    compiled=result.node.compiled_code,
-                    is_inline=is_inline,
-                    output_format=output_format,
-                    unique_id=result.node.unique_id,
-                )
+
+            compiled_node_event = CompiledNode(
+                node_name=result.node.name,
+                compiled=result.node.compiled_code,
+                is_inline=is_inline,
+                output_format=output_format,
+                unique_id=result.node.unique_id,
+                quiet=get_flags().QUIET,
             )
 
+            if get_flags().LOG_FORMAT == "json":
+                fire_event(compiled_node_event)
+            else:
+                # Cleaner to leave as print than to mutate the logger not to print timestamps.
+                print(compiled_node_event.message())
```

Comment:
We can use the new PrintEvent being added to dbt-common.

Reply:
Awesome! I'll check out https://github.com/dbt-labs/dbt-common/pull/130 and give `PrintEvent` a try.

Reply:
Note to self: see https://github.com/dbt-labs/dbt-core/pull/10131 for an example how to use `PrintEvent`.